R/get_generic.R

```diff
@@ -0,0 +1,72 @@
+## function name get_generic
+
+#' retrieve all parameters of specified type
+#' @param data parameter dataframe outputted from load_epidata()
+#' @param parameter_name name of the parameter type to retrieve, ensuring the name matches that in data
+#' @return dataframe with all parameter estimates and specified columns
```

Comment:
Yes, I think this is a good suggestion! I will add an issue so that mulitple people can work on this. For now, I have commented out the part that selects specific columns and just outputs the entire df but would be good for this to be a subset dependent upon pathogen and/or parameters.